dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/SpringBindingTest.java

```diff
@@ -135,6 +135,22 @@ void shouldReturnABadRequestWhenInvalidValueForBooleanInBindingObjectIsPassedAsP
                         "Value 'INVALID' is not valid for parameter booleanValue. Invalid boolean value [INVALID]")));
   }
 
+  @Test
+  void shouldReturnADefaultValueWhenDefaultParameterIsNotPassed() throws Exception {
+    mockMvc
+        .perform(get(ENDPOINT + "/default").param("booleanValue", "true"))
```

Comment:
Why do you pass `booleanValue` `true`? You are not asserting on anything it has an influence, right? If that is the case I would remove it.